translations/ja.json

```
@@ -547,70 +547,70 @@
-	"Specifying extra disks is currently only supported for the following drivers: {{.supported_drivers}}. If you can contribute to add this feature, please create a PR.": "",
-	"StartHost failed, but will try again: {{.error}}": "",
-	"Starting control plane node {{.name}} in cluster {{.cluster}}": "コントロールプレーンのノード {{.name}} を {{.cluster}} 上で起動しています",
-	"Starting minikube without Kubernetes {{.name}} in cluster {{.cluster}}": "",
-	"Starting tunnel for service {{.service}}.": "",
-	"Starting worker node {{.name}} in cluster {{.cluster}}": "",
-	"Starts a local Kubernetes cluster": "ローカル Kubernetes クラスタを起動します",
-	"Starts a local kubernetes cluster": "ローカル Kubernetes クラスタを起動します",
-	"Starts a node.": "",
-	"Starts an existing stopped node in a cluster.": "",
-	"Startup with {{.old_driver}} driver failed, trying with alternate driver {{.new_driver}}: {{.error}}": "",
-	"Stopped tunnel for service {{.service}}.": "",
-	"Stopping node \"{{.name}}\"  ...": "ノード \"{{.name}}\" を停止しています...",
-	"Stopping tunnel for service {{.service}}.": "サービス {{.service}} のトンネルを停止しています。",
-	"Stops a local Kubernetes cluster. This command stops the underlying VM or container, but keeps user data intact. The cluster can be started again with the \"start\" command.": "",
-	"Stops a node in a cluster.": "",
-	"Stops a running local Kubernetes cluster": "ローカル Kubernetes クラスタを停止します",
-	"Successfully added {{.name}} to {{.cluster}}!": "",
+	"Set failed": "設定に失敗しました",
+	"Set flag to delete all profiles": "全プロファイルを削除します",
+	"Set flag to stop all profiles (clusters)": "全プロファイル (クラスター) を停止します",
+	"Set flag to stop cluster after a set amount of time (e.g. --schedule=5m)": "設定時間後にクラスターを停止します (例: --schedule=5m)",
+	"Set this flag to delete the '.minikube' folder from your user directory.": "あなたのユーザーディレクトリー中の '.minikube' フォルダーを削除します。",
+	"Sets an individual value in a minikube config file": "minikube 設定ファイルの個別の値を設定します",
+	"Sets the PROPERTY_NAME config value to PROPERTY_VALUE\n\tThese values can be overwritten by flags or environment variables at runtime.": "PROPERTY_NAME の設定値を PROPERTY_VALUE に設定します\n\tこれらの値はランタイムのフラグ又は環境変数で上書きできます。",
+	"Sets up docker env variables; similar to '$(docker-machine env)'.": "docker 環境変数を設定します。'$(docker-machine env)' と同様です。",
+	"Sets up podman env variables; similar to '$(podman-machine env)'.": "podman 環境変数を設定します。'$(podman-machine env)' と同様です。",
+	"Setting profile failed": "プロファイルの設定に失敗しました",
+	"Show a list of global command-line options (applies to all commands).": "(全コマンドに適用される) 全般のコマンドラインオプションの一覧を表示します。",
+	"Show only log entries which point to known problems": "既知の問題を示すログエントリーのみ表示します",
+	"Show only the most recent journal entries, and continuously print new entries as they are appended to the journal.": "直近のジャーナルエントリーのみ表示し、ジャーナルに追加された新しいエントリーを連続して表示します。",
+	"Simulate numa node count in minikube, supported numa node count range is 1-8 (kvm2 driver only)": "minikube 中の NUMA ノードカウントをシミュレートします (対応 NUMA ノードカウント範囲は 1-8 (kvm2 ドライバーのみ)",
```

There is a unclosed parenteses`)` for first parenteses `(`.
```suggestion
	"Simulate numa node count in minikube, supported numa node count range is 1-8 (kvm2 driver only)": "minikube 中の NUMA ノードカウントをシミュレートします (対応 NUMA ノードカウント範囲は 1-8 (kvm2 ドライバーのみ))",
```